docs/modules/user-guide/partials/proc_migrating-commands.adoc

```diff
@@ -2,33 +2,25 @@
 = Migrating commands
 
 [role="_abstract"]
-This section describes how to migrate plug-ins to devfile v2. There ARE major changes in the `commands` section of the devfile. A command specified in a v1.0 devfile will NOT work as it is in a devfile v2.0. 
-
-.Procedure
-
-. Command Groups: build,run,test,debug. 
-. Apply Command. 
-. Environment Varibables for a Specific Command. 
-. Renaming workdir into workingDir. 
-. Id and label for Composite Commands. 
-. Run exec Commands as Specific User. 
-. Devfile metadata: add a link to an external website.
-. Stack/Devfile Matching Rules. 
+This section describes how to migrate commands to v2.0 devfiles. A command specified in a v1.0 devfile no longer works in a v2.0 devfile.
 
 [role="_additional-resources"]
 .Additional resources
 
-* Command Groups: build,run,test,debug. See https://github.com/che-incubator/devworkspace-api/issues/27[corresponding issue].
-* Apply Command. See https://github.com/devfile/api/issues/56[corresponding issue].
+To learn more about commands in v2.0 devfiles, go to the commands' GitHub issues on the following topics:
+
+* link:https://github.com/che-incubator/devworkspace-api/issues/27[Command groups: build,run,test,debug]
```

Comment:
NOTE: This comment applies to all links in this doc. The repo location has been changed. All links in this doc with prefix `https://github.com/che-incubator/devworkspace-api/issues/` should be changed to use the prefix `https://github.com/devfile/api/issues/`, e.g. change `https://github.com/che-incubator/devworkspace-api/issues/27` to `https://github.com/devfile/api/issues/27`.